File: frontend/test_fbclid.js
const { chromium } = require('playwright');
(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage();
  page.on('console', msg => {
    if (msg.type() === 'error') console.log('BROWSER ERROR CONSOLE:', msg.text());
  });
  page.on('pageerror', error => console.error('BROWSER ERROR STACK:', error.stack));
  await page.goto('http://localhost:3000/shared/sess_1768861489037?fbclid=IwY2xjawQIUrBleHRuA2FlbQIxMQBzcnRjBmFwcF9pZBAyMjIwMzkxNzg4MjAwODkyAAEe7eRhCnaWkVGb_OhL-O9GKLMfyDZqh-80jvkw6TVrVyZakZzVNrd6aqoUl1Y_aem_GF4y7fehig6GwDyvJFZU0A');
  await page.waitForTimeout(5000);
  await browser.close();
})();

File: test_fbclid.py
from playwright.sync_api import sync_playwright

def main():
    with sync_playwright() as p:
        browser = p.chromium.launch()
        page = browser.new_page()
        
        def handle_console(msg):
            print(f"BROWSER CONSOLE: {msg.type}: {msg.text}")
            
        def handle_error(err):
            print(f"BROWSER ERROR: {err}")
            
        page.on("console", handle_console)
        page.on("pageerror", handle_error)
        
        url = "http://127.0.0.1:3000/shared/sess_1768861489037?fbclid=IwY2xjawQIVz9leHRuA2FlbQIxMQBzcnRjBmFwcF9pZBAyMjIwMzkxNzg4MjAwODkyAAEe4SsdHjVT07Jd1QfXLElXFP7VeuGlhZqA_3meOjTpxDcNEoIUxrEFccI-Vis_aem_PdLa5N0ZFMceNQEJ1hUfBg"
        print(f"Navigating to {url}")
        try:
            page.goto(url, wait_until="networkidle")
        except Exception as e:
            print(f"Navigation exception: {e}")
            
        page.wait_for_timeout(3000)
        browser.close()

if __name__ == "__main__":
    main()
